drivers/adc-dac/ad5592r/ad5593r.c

```diff
@@ -3,7 +3,7 @@
  *   @brief  Implementation of AD5593R driver.
```

Comment:
Can you mention in the commit message that this is done to conform with the new platform driver?